### PR TITLE
test(known sapling contract): fix sapling known contract code accordi…

### LINKED ISSUE #1542: Taquito Integration Tests to Protocol J -  rpc-nodes.spec.ts - Executes tzip4 views by calling runView fails
On Monday net in the test suite 

 rpc-nodes.spec.ts

the test  Executes tzip4 views by calling runView fails with

```
    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 1

      Object {
        "data": Object {
    -     "int": "100",
    +     "int": "1",
        },
      }

      108 |           const views = await Tezos.rpc.runView(params)
      109 |           expect(views).toBeDefined();
    > 110 |           expect(views).toEqual({"data": {"int": "100"}});
          |                         ^
      111 |           done();
      112 |         });
      113 |

      at rpc-nodes.spec.ts:110:25
      at fulfilled (rpc-nodes.spec.ts:5:58)
```

### PR DESCRIPTION
**Pull request by roxaneletourneau**
…ng to the J protocol

Until fixes are made in the LocalForger, the RPC forger needs to be used to deploy a contract having
sapling on Mondaynet.

I also changed the initial storage of the knowBigMapContract to fix #1542 at the same time.

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/config.ts [hunk 1]
```diff
@@ -205,10 +201,12 @@ const faucetKeyFile = process.env['TEZOS_FAUCET_KEY_FILE'];
 const setupForger = (Tezos: TezosToolkit, forger: ForgerType): void => {
   if (forger === ForgerType.LOCAL) {
     Tezos.setProvider({ forger: localForger });
-  } else if (forger === ForgerType.COMPOSITE) {
+  } if (forger === ForgerType.COMPOSITE) {
```
Comment (dsawali) [inline integration-tests/config.ts:204]: I think we should put this in the next line since it's now a different if scope

Comment (roxaneletourneau) [reply integration-tests/config.ts]: Oops didn't mean to remove it, I put the else back, thanks.

File: integration-tests/originate-known-contracts.ts [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
 (async () => {
-    await tezosConfig.setup(true);
+    await setup(true);
     
     keyPkh = await tezos.signer.publicKeyHash();
     keyInitialBalance = await tezos.tz.getBalance(keyPkh);
 
     // KnownContract
-    await originateKnownContract('Contract', tezos, {
+     await originateKnownContract('Contract', tezos, {
```
Comment (dsawali) [inline integration-tests/originate-known-contracts.ts:24]: might be an accidental extra space here

**Review by danielelisi — APPROVED** [review]
LGTM; thanks for refactoring the script too!

File: integration-tests/originate-known-contracts.ts (hunk 2 above)
Comment (roxaneletourneau) [reply integration-tests/originate-known-contracts.ts]: fixed, thanks!

**Review by dsawali — APPROVED** [review]

**Pull request merged**